targets/targets.json

```diff
@@ -1451,7 +1451,14 @@
             "PSA"
         ],
         "is_disk_virtual": true,
-        "macros": ["CPU_MK64FN1M0VMD12", "FSL_RTOS_MBED", "MBEDTLS_PSA_CRYPTO_C"],
+        "macros": [
+            "CPU_MK64FN1M0VMD12",
+            "FSL_RTOS_MBED",
+            "MBEDTLS_PSA_CRYPTO_C",
+            "MBEDTLS_ENTROPY_NV_SEED",
```

Comment:
This device has TRNG enabling nvseed is redundant

Reply:
Pelion require injection API regardless TRNG